recipe/meta.yaml

```diff
@@ -24,8 +24,6 @@ test:
   commands:
     - test -f $PREFIX/include/benchmark/benchmark.h  # [unix]
     - if not exist %LIBRARY_INC%\\benchmark\\benchmark.h exit 1  # [win]
-    - test -f $PREFIX/lib/libbenchmark.a  # [unix]
```

Comment:
Make this `test ! -f` so that we check that the static lib is not accidentally included again.

Reply:
done